### DISCUSSION TITLE
how to use pixijs make https://github.com/zz85/threejs-path-flow

### DISCUSSION THREAD

**Discussion by a6965921** [pixijs/pixijs#8122] [Q&A]
how to use pixijs make https://github.com/zz85/threejs-path-flow   i want to make 2d anim

**Comment by domis86** [comment]
1. What exactly do you want to achieve? Can you show some sample 2d animation?
2. What have you tried already?

**Comment by ivanpopelyshev** [comment]
Make a plugin that does it, or request someone to make it for $$$ :) I know how path flow works, i dont see why it cant be coded for pixi
